buildscripts/cspi-mgmt/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -ex
+
+rm /usr/local/bin/zrepl
+exec /usr/local/bin/cspi-mgmt start
+exec service ssh start
```

Comment:
Should we start ssh service? cc: @vishnuitta

Reply:
lets have ssh.. and we may need to fix the ssh login as well in next PR..